src/content/docs/dropins/payment-services/containers/credit-card.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: CreditCard container
+description: Configure the CreditCard container to manage and display the available credit card fields during a checkout.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
+The `CreditCard` container allows a shopper to place an order with credit card details during the checkout process. You can configure it to manage and display the available credit card fields for a payment method.
+
+## CreditCard configurations
+
+The `CreditCard` container provides the following configuration options:
+
+<OptionsTable
+  compact
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['apiUrl', 'string', 'Yes', 'The URL to the Adobe Commerce GraphQL endpoint, such as "https://example.com/graphql".'],
+    ['getCartId', 'function', 'Yes', 'Should return a promise that resolves to the shopper`s cart ID.'],
+    ['getCustomerToken', 'function', 'No', 'The `CreditCard` container may send GraphQL requests on behalf of the shopper. This requires GraphQL authorization, which can be performed using authorization tokens or session cookies. For token-based authorization, the "getCustomerToken" function that should return a customer token as a string, or null for guest checkouts.'],
+    ['onSuccess', 'function', 'No', 'Called when payment flow finishes successfully. This function is not passed any arguments.'],
+    ['onError', 'function', 'No', 'Called when the payment flow was aborted due to an error. The function receives a single argument, {message: string}, containing the reason of error.'],
+    ['onRender', 'function', 'No', 'Called when the credit card fields render successfully. The function receives the credit card instance as its single argument, which has a submit() method to be used to start the payment flow.'],
+    ['onValidation', 'function', 'No', 'Called when the credit card form validity changes. Use this to disable/enable a "Place order" button.'],
+  ]}
+/>
+
+## Example
+
+The following example renders the `CreditCard` container to handle and display the available credit card fields as a payment method.
+
+```js
+import CreditCard, { CREDIT_CARD_CODE } from '@dropins/payment-services/containers/CreditCard.js';
+import { render as PaymentServicesProvider } from '@dropins/payment-services/render.js';
+import { getConfigValue } from '../../scripts/configs.js';
+import { getUserTokenCookie } from '../../scripts/initializers/index.js';
+
+const apiUrl = await getConfigValue('commerce-core-endpoint');
+const container = document.createElement('div');
+const placeOrderButton = document.getElementById('place-order');
+const getCartId = () => 'ozGi7uLI74etDYyMijoI2cla5CmGIBch';
+
+PaymentServicesProvider.render(CreditCard, {
+  apiUrl: apiUrl,
+  getCustomerToken: getUserTokenCookie,
```

Comment:
Before merging, we should make sure that `getUserTokenCookie` actually returns `null` for guest checkouts. If not, we can provide a lambda, à la `() => async getUserTokenCookie() || null`.